sdk/storage/Azure.Storage.Blobs/src/LeaseClient.cs

```diff
@@ -131,10 +136,10 @@ private void EnsureClient()
         /// For more information, see <see href="https://docs.microsoft.com/rest/api/storageservices/lease-container" />.
         /// </summary>
         /// <param name="duration">
-        /// Specifies the duration of the lease, in seconds, or -1 for a lease
-        /// that never expires.  A non-infinite lease can be between 15 and
-        /// 60 seconds. A lease duration cannot be changed using
-        /// <see cref="RenewAsync"/> or <see cref="ChangeAsync"/>.
+        /// Specifies the duration of the lease, in seconds, or specify 
+        /// LeaseClient.InfiniteLeaseDuration for a lease that never expires.  
```

Comment:
Nit - you can change this to just `InfiniteLeaseDuration` and then hit `Ctrl+.` in stock Visual Studio to pop-up the fix that will wrap it in a ref automatically so it hyperlinks for users in the docs.